cmd/asset/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM golang:1.12.10-alpine3.10 as godev
+
+# alpine-sdk includes git, make, etc.
+RUN apk add \
+    --repository https://dl-3.alpinelinux.org/alpine/edge/testing \
+    --repository https://dl-3.alpinelinux.org/alpine/edge/main \
+    --repository https://dl-3.alpinelinux.org/alpine/edge/community \
+    --update \
+    --no-cache \
+    alpine-sdk vips vips-dev
+
+ENV GO111MODULE on
+
+WORKDIR /go/src/github.com/skygeario/skygear-server
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+RUN cd cmd/asset; make build; mv skygear-asset /tmp/
+
+FROM alpine:3.10
+
+RUN apk add \
+    --repository https://dl-3.alpinelinux.org/alpine/edge/testing \
+    --repository https://dl-3.alpinelinux.org/alpine/edge/main \
+    --repository https://dl-3.alpinelinux.org/alpine/edge/community \
+    --update \
+    --no-cache \
+    libc6-compat ca-certificates vips vips-dev && \
```

Comment:
It seems `libc6-compat` and `vips-dev` are not needed.

Reply:
libc6-compat is not added by me.
I guess vips-dev is needed because this package provides header files.

Reply:
`vips-dev` is needed at build time, but it shouldn't be needed at runtime?

Reply:
From [the description of libc6-compat](https://pkgs.alpinelinux.org/package/edge/main/x86/libc6-compat), it seems that it is needed if the program uses glibc. And vips does use glibc.

Reply:
Anyway let me experiment if we can remove both

Reply:
`ldd` seems to show it's using musl:
```
/ # ldd skygear-asset 
        /lib/ld-musl-x86_64.so.1 (0x7f68c629b000)
        libvips.so.42 => /usr/lib/libvips.so.42 (0x7f68c5e1a000)
        libgobject-2.0.so.0 => /usr/lib/libgobject-2.0.so.0 (0x7f68c5dd0000)
        libglib-2.0.so.0 => /usr/lib/libglib-2.0.so.0 (0x7f68c5cd1000)
        libc.musl-x86_64.so.1 => /lib/ld-musl-x86_64.so.1 (0x7f68c629b000)
...
```